core/src/main/java/io/seata/core/rpc/netty/AbstractNettyRemoting.java

```diff
@@ -209,6 +219,9 @@ protected void sendAsync(Channel channel, RpcMessage rpcMessage) {
             LOGGER.debug("write message:" + rpcMessage.getBody() + ", channel:" + channel + ",active?"
                 + channel.isActive() + ",writable?" + channel.isWritable() + ",isopen?" + channel.isOpen());
         }
+
+        doBeforeRpcHooks(ChannelUtil.getAddressFromChannel(channel), rpcMessage);
```

Comment:
Where is the corresponding `doAfterRpcHooks ` ?

Reply:
There is no way to locate the result of the asynchronous message, so it is not added. Of course, only the send is added. It is the same for each address and does not affect the result.